Source/DafnyLanguageServer/Language/DafnyProgramVerifier.cs

```diff
@@ -158,14 +174,36 @@ private class ModelCapturingOutputPrinter : OutputPrinter {
 
       public void Inform(string s, TextWriter tw) {
         logger.LogInformation(s);
-        var match = Regex.Match(s, "^Verifying .+[.](?<name>[^.]+) [.][.][.]$");
-        if (match.Success) {
-          progressReporter.ReportProgress(match.Groups["name"].Value);
-        }
```

Comment:
This code is no longer useful because verification diagnostics take care of reporting progress. Unless we deactivate it...?